game_data.json

```diff
@@ -0,0 +1 @@
+{"width_slider": 15, "height_slider": 10, "difficulty_slider": 15, "high_scores": [{"time": 6, "width": 10, "height": 10, "mine_count": 5}]}
```

Comment:
This file should be deleted. I think you committed it accidentally. The latest `main` branch has it in `.gitignore` so that you can't commit it accidentally.